pkg/util/resource.go

```diff
@@ -73,5 +74,14 @@ func GetPodEffectiveRequest(pod *v1.Pod) v1.ResourceList {
 		}
 		resources[name] = quantity
 	}
+	// Add pod overhead if present
+	if pod.Spec.Overhead != nil {
```

Comment:
Overhead is a map, so we can directly iterate it w/o this if check.

Reply:
done